src/main/java/gift/domain/point/PointRestController.java

```diff
@@ -0,0 +1,37 @@
+package gift.domain.point;
+
+import gift.domain.member.dto.LoginInfo;
+import gift.global.resolver.Login;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+
+@RestController
+@Tag(name = "Point", description = "Point API")
+public class PointRestController {
+    private final PointService pointService;
+    public PointRestController(PointService pointService) {
+        this.pointService = pointService;
+    }
+
+    @GetMapping("/api/points")
+    @Operation(summary = "내 포인트 조회")
+    public ResponseEntity<PointRequestResponse> getPoint(@Login LoginInfo loginInfo) {
+        int point = pointService.getPoint(loginInfo.getId());
+        return ResponseEntity.ok(new PointRequestResponse(point));
+    }
+
+    @PostMapping("/api/points")
```

Comment:
POST로 충전을 기대하기는 어려워서요ㅎㅎ 다음과 같이 path 정하는 것도 괜찮았을 것 같네요! 
```suggestion
    @PostMapping("/api/points/charge")
```

Reply:
생각해보니 POST 가 충전의 의미와는 먼 것 같습니다. 프론트엔드와 협의해보겠습니다. 말씀 감사합니다 ㅎㅎ